### ISSUE TITLE
Google pay type changed from "paywithgoogle" to "googlepay"

### ISSUE THREAD

**Issue by jorismathijssen** [Adyen/adyen-dotnet-api-library#657]
**Describe the bug**
The JSON that is generated inside the Android SDK for GooglePay has changed from "paywithgoogle" to "googlepay" [here](https://github.com/Adyen/adyen-android/blob/aaae6c5bcfdca48980d36a350d32f47eef92169b/components-core/src/main/java/com/adyen/checkout/components/util/PaymentMethodTypes.kt#L30). However this is not supported by the C# JsonConverters. These converters still check for the "paywithgoogle" type

**To Reproduce**
1. Generate a Android SDK google pay response with type "googlepay"
2. Use this json and check if its converted to a GooglePayDetails object

**Expected behavior**
The JSON should be deserialized into a GooglePayDetails object.

**Smartphone (please complete the following information):**
 - Device: Android
 - Drop-In
 - 4.8.0

**Additional context**

**Comment by jorismathijssen** [comment]
I personally don't understand why the Android SDK has changed this (quite some time ago) but there has been no changes in this and the iOS SDK?

**Comment by wboereboom** [comment]
Hi @jorismathijssen,

Thanks for raising this issue & contributing.
The Adyen API is constantly changing and evolving. This issue you touch upon is an example of such a change.
Up until now we have not had a good answer to update our API libraries based on these changes. 
The result of this are libraries with outdated models.
Fortunately we are working hard on automating all model updates, so our libraries will never be out of date, and if they are, we can update them within the day.
We have already rolled this out for our NodeJS library, and we are currently in the process of doing the same for Java.
.Net is up next!

tl;dr: Our library models are (unfortunately) out of date, but we are going to automate model updates in the near future so confusing stuff like this doesn't happen anymore.

Kind Regards,
Wouter
Adyen

**Comment by jorismathijssen** [comment]
> Hi @jorismathijssen,
> 
> Thanks for raising this issue & contributing. The Adyen API is constantly changing and evolving. This issue you touch upon is an example of such a change. Up until now we have not had a good answer to update our API libraries based on these changes. The result of this are libraries with outdated models. Fortunately we are working hard on automating all model updates, so our libraries will never be out of date, and if they are, we can update them within the day. We have already rolled this out for our NodeJS library, and we are currently in the process of doing the same for Java. .Net is up next!
> 
> tl;dr: Our library models are (unfortunately) out of date, but we are going to automate model updates in the near future so confusing stuff like this doesn't happen anymore.
> 
> Kind Regards, Wouter Adyen

This is the perfect answer! :) That would be a fantastic solution!
